include/swift/Option/Options.td

```diff
@@ -554,6 +554,10 @@ def dump_parse : Flag<["-"], "dump-parse">,
 def dump_ast : Flag<["-"], "dump-ast">,
   HelpText<"Parse and type-check input file(s) and dump AST(s)">, ModeOpt,
   Flags<[FrontendOption, NoInteractiveOption, DoesNotAffectIncrementalBuild]>;
+def dump_serialized_syntax_tree : Flag<["-"], "dump-serialized-syntax-tree">,
```

Comment:
What do you get from the word "serialized"?

Alternately, is the word "dump" correct, if this is actually a stable format?